src/thriveopedia/fossilisation/MuseumCard.cs

```diff
@@ -49,12 +64,16 @@ public class MuseumCard : Button
         }
     }
 
+    public string? FossilName { get; set; }
```

Comment:
We currently check for fossilised species overwriting by comparing file names, so I'm wary that using the species name instead for deleting could lead to unexpected behaviour if file names have changed to have duplicates. Admittedly that's a very unlikely scenario and a quick test suggests it doesn't cause any problems.

Maybe it's worth changing to always compare species names. That doesn't have to be done in this PR though.

Reply:
> I'm wary that using the species name instead for deleting could lead to unexpected behaviour if file names have changed to have duplicates

I'm not sure I understand. This PR specifically uses the file name for deleting. If that's unclear I could rename the property to FileName or something?

Reply:
Ah, my bad, I forgot FossilisedSpecies.Name refers to the file name rather than the species name. Don't worry about it.